lib/oscap_tests.pm

```diff
@@ -123,186 +711,541 @@ sub upload_logs_reports {
     };
 =cut
 
-sub oscap_security_guide_setup {
+sub get_tests_config {
+    # Get the tests configuration file from repository
+    my $config_file_name = "openqa_config.conf";
+    my $url = "https://gitlab.suse.de/seccert-public/compliance-as-code-compiled/-/raw/main/content/";
+
+    download_file_from_https_repo($url, $config_file_name);
+
+    my $config_file_path = script_output("pwd", quiet => 1);
+    $config_file_path =~ s/\r|\n//g;
+    $config_file_path .= "/$config_file_name";
+
+    my $data = script_output("cat $config_file_path", quiet => 1);
+    my $config = Config::Tiny->new;
+    $config = Config::Tiny->read_string("$data");
+    my $err = Config::Tiny::errstr;
+    if ($err eq "") {
```

Comment:
skip the error handling, if it fails it should die and a stack trace is printed anyway in openQA, so I think it is not needed, also notice typo `s/returened/returned/`.
Additionally I think the return value is not used anywhere, but if you pick this from data/ you would not even need this function.

Reply:
Fixed typo.
I think error handling is valid here.
In case of phrasing error it will use default values.